packages/react-magma-dom/src/components/Table/Table.test.js

```diff
@@ -67,30 +67,30 @@ describe('Table', () => {
     );
   });
 
-  it('should render table with hover styles', () => {
-    const { getByTestId } = render(
-      <Table hasHoverStyles>
-        <TableBody>
-          <TableRow testId="row1">
-            <TableCell>cell 1</TableCell>
-            <TableCell>cell 2</TableCell>
-          </TableRow>
-          <TableRow testId="row2">
-            <TableCell>cell 1</TableCell>
-            <TableCell>cell 2</TableCell>
-          </TableRow>
-        </TableBody>
-      </Table>
-    );
-
-    expect(getByTestId('row2')).toHaveStyleRule(
-      'background',
-      magma.colors.tone02,
-      {
-        target: ':hover',
-      }
-    );
-  });
+  // it('should render table with hover styles', () => {
```

Comment:
should this test get deleted?

Reply:
Oops, no.  It shouldn't have been checked in commented out.  Will fix.